modules/preseq/ccurve/main.nf

```diff
@@ -0,0 +1,40 @@
+process PRESEQ_CCURVE {
+    tag "$meta.id"
+    label 'process_medium'
+    label 'error_ignore'
```

Comment:
haven't seen this yet, where does it come from?

Reply:
The `error_ignore`? 

https://github.com/nf-core/rnaseq/blob/5aadd93370c836e12b777057eb2298fc7bc24fc9/conf/base.config#L43

Useful for qc checks that may fail, but you don't want to exit the whole workflow. I don't know why @drpatelh included it originally.

Reply:
ah ok, never saw it :)